test/scripts/e2e-kind.sh

```diff
@@ -170,10 +170,17 @@ export NUM_WORKER_NODES=3
 if [ "$SINGLE_NODE_CLUSTER" == true ]; then
 	export NUM_WORKER_NODES=1
 fi
+
+# Until we know how to make github actions gracefully terminate our tests, this
+# timeout needs to be lower than github's timeout. Otherwise github terminates
```

Comment:
is github's timeout: https://github.com/ovn-org/ovn-kubernetes/blob/8551af5a88ee86169b433d9b2edf304440a02438/.github/workflows/test.yml#L530 120mins? hence 100mins selected to be lower?